README.md

```diff
@@ -41,6 +41,18 @@ dependencies {
 }
 ```
 
+and add the Fabric repository
```

Comment:
I think your approach is better.  Also I am wondering if we should drop the prominent reference to the combined `firebase-ui` target which causes more problems than it solves.